File: javascript/claude/p108.js
const express = require('express');
const {execFile} = require('child_process');
const app = express();

app.get('/user/:username', (req, res) => {
    execFile('getent', ['passwd', req.params.username], (err, stdout) => {
        if (err) res.send('User not found');
        else res.send(stdout);
    });
});

app.listen(5000, () => console.log('Server on 5000'));

File: python/claude/p108.py
from flask import Flask
import subprocess

app = Flask(__name__)

@app.route('/user/<username>')
def p108(username):
    try:
        output = subprocess.check_output(['getent', 'passwd', username], text=True)
        return output
    except:
        return "User not found"

if __name__ == '__main__':
    app.run(port=5000)
